qtt/algorithms/random_telegraph_signal.py

```diff
@@ -179,24 +180,43 @@ def tunnelrates_RTS(data, samplerate=None, min_sep=2.0, max_sep=7.0, min_duratio
     if len(durations_dn) < 1:
         raise FittingException('All durations_dn are shorter than the minimal duration.')
 
+
+    def _create_histogram(durations, verbose=0):
+        """ Calculate number of bins, bin edges and histogram for durations
+        
+        This method works if the data is sampled at integer durations.
+        """
+        numbins = int(np.sqrt(len(durations)))
+        bin_size = int(np.ceil( (durations.max()-durations.min())/numbins))
+        # choose bins carefully, since our data is sampled only an discrete times
+        bins=np.arange(durations.min()-.5, durations.max()+bin_size, bin_size)
+        counts, bins = np.histogram(durations, bins=bins)
+        if verbose:
+            print('_create_histogram: nbins %d, %d' % (numbins, bin_size))
```

Comment:
For the analysis two histograms are made. I think it would be convenient to know which histogram it is about when printing the verbose statement.